hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/node/TestDecommissionAndMaintenance.java

```diff
@@ -343,21 +324,8 @@ public void testSingleNodeWithOpenPipelineCanGotoMaintenance()
     DatanodeDetails newDN = nm.getNodeByUuid(dn.getUuid().toString());
     waitForDnToReachHealthState(newDN, HEALTHY);
     waitForDnToReachPersistedOpState(newDN, IN_MAINTENANCE);
-  }
-
-  @Test
-  // After a node enters maintenance and is stopped, it can be recommissioned in
-  // SCM. Then when it is restarted, it should go back to IN_SERVICE and have
-  // that persisted on the DN.
-  public void testStoppedMaintenanceNodeTakesScmStateOnRestart()
```

Comment:
This test is merged into `testSingleNodeWithOpenPipelineCanGotoMaintenance` by recommissioning after maintenance is complete.